src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,13 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Returns if the command mutates the data.
+     * True is the dafault return.
+     * @return true if the commands mutates the data; false if not
+     */
+    public boolean isMutating() {
+        return true;
```

Comment:
Though make a default behaviour for parent class can save an amount of lines of code, it might not follow the intuitiveness of OOP. Could we spare this part in **Command** and ask every child of it must reimplement?

Reply:
I see. But if I do not include the return statement, it will be a compilation error. So should I make it an abstract method in **Command** class here?

Reply:
Sorry for my late reply. Yes, abstract is what I'm talking about here 👍

Reply:
To be clear, it's great if you can update it. However, I can say safely say you satisfy this LO.